packages/contentful/migrations/crn/researchOutputs/20231012113418-change-created-date-to-auto-populate-field.js

```diff
@@ -0,0 +1,20 @@
+module.exports.description = 'Change createdDate field editor';
+
+module.exports.up = (migration) => {
+  const researchOutputs = migration.editContentType('researchOutputs');
+  researchOutputs.changeFieldControl(
```

Comment:
The field still appears in the UI though and it seems it's a requirement of the ticket to hide it. Maybe we'll need to select this "Hide field when editing"

<img width="275" alt="Screenshot 2023-10-17 at 06 31 15" src="https://github.com/yldio/asap-hub/assets/16595804/3aaae646-ab1d-41ea-b260-d1e87dc07f0b">

It seems what it changes in the field is that it changes this

```
"disabled": true,
```

Reply:
If you hide it while editing the field is not in the form, therefore it does not auto populate so the `createdDate ` field is set to null which results in the FE showing date as 01-01-1970